### PR TITLE
Disease fixes

### LINKED ISSUE #36178: Players unable to ghost out of sentient disease antag
Okay round's over, honest mistake!

[Round ID]: # 84532

[Reproduction]: Triggered the new Sentient Disease antag event to get a better look at it, the player who received it played for a while and it all went fine, disease was eventually purged except for one last host who was a braindead and isolated atmos tech. The player controlling the disease ahelped after five-ten minutes to report that he was unable to ghost out of his disease role and I wound up having to VV delete him so he could exit his braindead host. Not sure if it's really a bug or just an oversight, but it seems like some kind of non-admin contingency needs to be available for a player virus who ends up stuck in a non-productive host.

### LINKED ISSUE #36183: all my diseases are colds
I started a virus from a cold bottle and now I can't rename anything at all, everything is still a cold, even after Sneezing was removed.

### PR DESCRIPTION
**Pull request by Cruix**
Stuff like this is why I wanted #35988 to be testmerged first.

Fixes #36183 
Fixes #36178

### CODE REVIEW

**Comment by AnturK** [comment]
I was informed it was testmerged already, my bad for not checking.

File: code/modules/reagents/chemistry/machinery/pandemic.dm [hunk 1]
```diff
@@ -57,7 +57,7 @@
 		if(istype(D, /datum/disease/advance))
 			var/datum/disease/advance/A = D
 			var/disease_name = SSdisease.get_disease_name(A.GetDiseaseID())
-			if(disease_name == "Unknown" && A.mutable)
+			if((disease_name == "Unknown") && A.mutable)
```
Comment (AnturK) [inline code/modules/reagents/chemistry/machinery/pandemic.dm]: == already has precedence here.

Comment (MoreRobustThanYou) [reply code/modules/reagents/chemistry/machinery/pandemic.dm]: it does look cleaner, though

**Comment by vuonojenmustaturska** [comment]
it WAS testmerged though

File: code/modules/reagents/chemistry/machinery/pandemic.dm (hunk 1 above)
Comment (Cruix) [reply code/modules/reagents/chemistry/machinery/pandemic.dm]: I did this for clarity, I am surprised I had not initially.

**Comment by davethwave** [comment]
Viro is rarely done as a job and there is only 1 viro per round so it's not surprising that there would be some bugs unnoticed

**Comment by Cruix** [comment]
I meant that wanted it to be testmerged long enough to receive feedback and push changes to the original PR.

**Pull request merged**